### ISSUE TITLE
Dev: Key error on startup

### ISSUE THREAD

**Issue by sti0** [jseidl/hass-magic_areas#33]
I'm using the dev branch but at startup the component fails if an area is not defined in yaml.

```
2020-10-25 19:54:45 INFO (MainThread) [custom_components.magic_areas] Creating area 'Garderobe' (#b05b9d2971184abc85dca7a607c3c5ae) with 9 entities.
2020-10-25 19:54:45 ERROR (MainThread) [homeassistant.setup] Error during setup of component magic_areas
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 213, in _async_setup_component
    result = await task
  File "/config/custom_components/magic_areas/__init__.py", line 92, in async_setup
    magic_area = MagicArea(
  File "/config/custom_components/magic_areas/__init__.py", line 149, in __init__
    self.config = config[self.slug]
KeyError: 'garderobe'
```
https://github.com/jseidl/hass-magic_areas/blob/8e64f46bab2edea69cb54a9be28bcf025e806bd3/custom_components/magic_areas/__init__.py#L149
This won't happen if I use the release 1.0.2.

**Comment by jseidl** [comment]
Indeed. `1.0.2` has defaults set and the one on `develop` just trusts the config. Need to add a check `if self.slug in config.keys()` and if not load default parameters.

Scheduled for `1.0.3`

**Comment by jseidl** [comment]
Fixed on `develop` branch with #41
